src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -965,11 +987,36 @@ void Subgraph::SubgraphExecutor::parallel_forNd(const std::function<void(jit_sni
                 tmp /= dom[j];
             }
 
-            caller(call_args, indexes.data());
+            caller(call_args, indexes);
         }
     });
 }
 
+void Subgraph::SubgraphExecutor::execute(dnnl::stream strm, const std::vector<MemoryPtr>& inMemPtrs, const std::vector<MemoryPtr>& outMemPtrs) {
+    if (!m_in_requested_descs.empty()) {
+        auto reorderedInMemPtrs = exec_in_reorders(strm, inMemPtrs);
```

Comment:
Minor: Probably, better to rename to `reorder_inputs`/`reorder_needed_inputs`?

Reply:
Done, thanks for the idea